Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Builds a Docker image with Firedrake and its dependencies:
+#
+#   http://firedrakeproject.org
+#
+# This image is hosted on Docker Hub:
+#
+#   https://registry.hub.docker.com/u/firedrakeproject/
+#
+# Author: Florian Rathgeber <florian.rathgeber@gmail.com>
+
+FROM phusion/baseimage:0.9.18
+MAINTAINER Florian Rathgeber <florian.rathgeber@gmail.com>
+
+# Install required packages (we cannot use the firedrake-install script for
+# this since we have no Python yet)
+RUN apt-get -qq update && \
+    apt-get -y install \
+      build-essential \
+      cmake \
+      gfortran \
+      git-core \
+      libblas-dev \
+      liblapack-dev \
+      libopenmpi-dev \
+      libspatialindex-dev \
+      mercurial \
+      openmpi-bin \
+      python-dev \
+      python-pip && \
```

Comment:
This causes a maintenance problem because we now have our dependency list written twice. Can we not just apt-get install python and let firedrake-install do the rest?